helper/DockerHelper.go

```diff
@@ -455,7 +455,7 @@ func executeCmd(dockerBuild string) error {
 }
 
 func tagDockerBuild(dockerRepository string, dest string) error {
-	dockerTag := "docker tag " + dockerRepository + ":latest" + " " + dest
+	dockerTag := fmt.Sprintf("docker tag %q:latest %q", dockerRepository, dest)
```

Comment:
please test this case

Reply:
Have test it. It updates the latest tag successfully.